index.js

```diff
@@ -283,32 +292,41 @@ app.get("/joinclass", async (req, res) => {
     classroom = await db.get(code);
   }
 
-  classroom.push(email);
-  db.set(code, classroom);
-
-
-
-  let studentsToCodes = {};
-
-  if (codeList.indexOf("studentsToCodes") > -1) {
-    studentsToCodes = await db.get("studentsToCodes");
+  var studentAlreadyHere;
+  if (codeList.includes(code)) {
+    studentAlreadyHere = true;
   }
+  else {
+    classroom.push(email);
+    db.set(code, classroom);
+    
+    let studentsToCodes = {};
```

Comment:
I think you're supposed to use var here instead of let, since otherwise studentsToCodes won't be defined later on